### DISCUSSION TITLE
Query on how the timeofday category was chosen

### DISCUSSION THREAD

**Discussion by HaoLin97** [bdd100k/bdd100k#180] [Q&A] [answered]
Hi @fyu ,
I'm just wondering how the timeofday category was labelled in the dataset. Was it categorized based on the 24hr clock or was it categorized based on the sun's alignment or labelled manually?

Any information would be appreciated.

**Comment by siyuanliii** [comment, marked as answer]
Please refer to this document: https://doc.bdd100k.com/instructions/tagging.html
